apps/smtp-server/src/server.ts

```diff
@@ -0,0 +1,100 @@
+import { SMTPServer, SMTPServerOptions } from 'smtp-server';
+import { Readable } from 'stream';
+import axios from 'axios';
+import dotenv from 'dotenv';
+import { simpleParser } from 'mailparser';
+
+dotenv.config();
+
+const AUTH_USERNAME = process.env.SMTP_AUTH_USERNAME!;
+const UNSEND_BASE_URL = process.env.UNSEND_BASE_URL!;
+let API_KEY = '';
+
+const ports = [25, 465, 2465, 587, 2587]; // Array of ports to listen on
```

Comment:
is there any documentation on how to handle certificates and stuff? just want to know how to implement them!

Reply:
I will need to check on that